stacks/apps/augury.yml

```diff
@@ -677,6 +680,78 @@ Resources:
         - MetricName: !GetAtt Constants.SlowJobErrorsCountMetricName
           MetricNamespace: !Ref kMetricFilterNamespace
           MetricValue: "1"
+  WebLoggedErrorsMetricFilter:
+    Type: AWS::Logs::MetricFilter
+    Properties:
+      FilterPattern: |-
+        { $.level >= 50 }
+      LogGroupName: !Ref WebTaskLogGroup
+      MetricTransformations:
+        - MetricName: !Sub Constants.WebLoggedErrorsMetricName
+          MetricNamespace: !Ref kMetricFilterNamespace
+          MetricValue: "1"
+  WorkersLoggedErrorsMetricFilter:
+    Type: AWS::Logs::MetricFilter
+    Properties:
+      FilterPattern: |-
+        { $.level >= 50 }
+      LogGroupName: !Ref WorkerTaskLogGroup
+      MetricTransformations:
+        - MetricName: !Sub Constants.WorkerLoggedErrorsMetricName
+          MetricNamespace: !Ref kMetricFilterNamespace
+          MetricValue: "1"
+  WorkersSlowLoggedErrorsMetricFilter:
+    Type: AWS::Logs::MetricFilter
+    Properties:
+      FilterPattern: |-
+        { $.level >= 50 }
+      LogGroupName: !Ref SlowWorkerTaskLogGroup
+      MetricTransformations:
+        - MetricName: !Sub Constants.SlowWorkerLoggedErrorsMetricName
+          MetricNamespace: !Ref kMetricFilterNamespace
+          MetricValue: "1"
+  AuguryWebLoggedErrorsAlarm:
+    Type: AWS::CloudWatch::Alarm
+    Properties:
+      AlarmName: !Sub ERROR [Augury] Web <${EnvironmentTypeAbbreviation}> IS LOGGING ERRORS (${RootStackName})
+      AlarmDescription: !Sub >-
+        ${EnvironmentType} Augury's web task has logged an error. Check the logs!
+      ComparisonOperator: GreaterThanOrEqualToThreshold
```

Comment:
I see in other error alarms we use `ComparisonOperator: GreaterThanThreshold`, this seems right here though, as it is consistent with other Augury alarms, and it will keep this from being a flood of slack messages, while the low threshold will alert on any error.